### PR TITLE
Restyle compensation interface

### PR DESCRIPTION
**Pull request by NoahSaso**
This PR:
- styles the UI like the familiar list of proposals
- adds a card background like the creation form to the nomination form

## Nothing created yet
Screenshot 2022-12-07 at 6 09 51 PM

## Upcoming cycle (in the fuuuuuuture)
Screenshot 2022-12-07 at 6 11 37 PM

## Submissions open, have not submitted yet
Screenshot 2022-12-07 at 6 12 18 PM

## Submission form visible after clicking on row
Screenshot 2022-12-07 at 6 12 38 PM

## Submissions open, submitted already
Screenshot 2022-12-07 at 6 13 31 PM

## Submissions closed, rating open, member of DAO
Screenshot 2022-12-07 at 6 14 24 PM

## Submissions closed, rating open, not member of DAO who did not submit
Screenshot 2022-12-07 at 6 15 44 PM
Screenshot 2022-12-07 at 6 16 17 PM

## Rating form visible after clicking on row (which prompted for auth upon click)
Screenshot 2022-12-07 at 6 17 55 PM

## Ratings closed, awaiting completion (pending proposal creation), member of DAO
image

## Ratings closed, awaiting completion (pending proposal creation), not member of DAO who did submit during intake
Screenshot 2022-12-07 at 6 19 07 PM

## Proposal created, cycle closed
Screenshot 2022-12-07 at 6 24 19 PM

### CODE REVIEW

File: packages/i18n/locales/en/translation.json [hunk 1]
```diff
@@ -486,7 +486,9 @@
     "stakingAddress": "Staking Contract",
     "subDaoExplanation": "SubDAOs are children DAOs that help manage groups within your DAO.",
     "subDaoNeedsAdding": "{{parent}} has not yet recognized {{child}} as a SubDAO.",
+    "submissionsBeingRated": "Members of the DAO are rating submissions. The DAO will publish a proposal shortly after they finish.",
```
Comment (elsehow) [inline packages/i18n/locales/en/translation.json:489]: Maybe:

> Members of the DAO are rating submissions. After the rating period closes, the DAO will publish a proposal to distribute compensation according to their responses.

File: packages/stateful/payroll/adapters/Retroactive/components/stateful/OpenSurveySection.tsx [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+      } = await postRequest(`/${coreAddress}/contributions`)
+
+      // Get addresses for contributor public keys.
+      const contributions = await Promise.all(
+        response.contributions.map(
+          async ({
+            contributor: publicKey,
+            ...contribution
+          }): Promise<Contribution> => {
+            const address = await secp256k1PublicKeyToBech32Address(
```
Comment (elsehow) [inline packages/stateful/payroll/adapters/Retroactive/components/stateful/OpenSurveySection.tsx:69]: Nice

File: packages/stateful/payroll/adapters/Retroactive/components/stateful/PayrollTab.tsx [hunk 3]
```diff
@@ -84,6 +46,29 @@ export const InnerPayrollTab = () => {
     ),
     []
   )
+  // Get voting power at time of each completed survey creation to determine if
+  // we can download the CSV or not.
```
Comment (elsehow) [inline packages/stateful/payroll/adapters/Retroactive/components/stateful/PayrollTab.tsx:50]: Nice, great comment.

**Review by elsehow — APPROVED** [review]

File: packages/i18n/locales/en/translation.json (hunk 1 above)
Comment (NoahSaso) [reply packages/i18n/locales/en/translation.json]: I like it.

**Pull request merged**